libzfs-sys/build.rs

```diff
@@ -8,6 +8,10 @@ extern crate pkg_config;
 use std::env;
 
 fn main() {
+    if cfg!(target_os = "macos") {
```

Comment:
Should we whitelist instead of blacklist?

Reply:
maybe, want some more feedback before deciding one way or another.